conf/ceres.config

```diff
@@ -0,0 +1,28 @@
+params {
+  config_profile_description = 'USDA ARS SCINet Ceres Cluster profile'
+  config_profile_contact = 'Thomas A. Christensen II (@MillironX)'
+  config_profile_url = 'https://scinet.usda.gov/guide/ceres/'
+
+  max_memory = 640.GB
+  max_cpus = 36
+  max_time = 60.d
+}
+
+singularity {
+  enabled = true
+  autoMounts = true
+}
+
+process {
+  executor = 'slurm'
+  scratch = true
+  queue = {
+    task.memory >= 216.GB ?
+      task.time >= 7.d ?
+        'longmem' : 'mem' :
```

Comment:
This will mean you will always default to `mem` (the subsequent `task.time`s would not reach evaluation).

Reply:
Redone with `switch` statements.